content/hi/nodes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: नोड्स (Nodes) 
+status: Completed
+category: संकल्पना
+exclude_search: true
+---
+
+## यह क्या है
+
+नोड एक कंप्यूटर है जो एक सामान्य कार्य को पूरा करने के लिए अन्य कंप्यूटरों या नोड्स के साथ मिलकर काम करता है। उदाहरण के लिए, अपना लैपटॉप, मॉडेम और प्रिंटर लें। वे सभी आपके वाई-फाई (Wi-Fi) नेटवर्क से जुड़े हुए हैं जो संचार और सहयोग कर रहे हैं, प्रत्येक एक नोड का प्रतिनिधित्व करता है। क्लाउड कंप्यूटिंग में, नोड एक भौतिक कंप्यूटर, एक वर्चुअल कंप्यूटर, जिसे VM (Virtual Machine) या एक कंटेनर भी कहा जाता है, हो सकता है।
```

Comment:
@starlightknown can you review this sentence again
`क्लाउड कंप्यूटिंग में, नोड एक भौतिक कंप्यूटर, एक वर्चुअल कंप्यूटर, जिसे वर्चुअल मशीन (Virtual Machine) या एक कंटेनर भी कहा जाता है, हो सकता है।`
 because I think the sentence will be incomplete without 
`हो सकता है।`

Reply:
In cloud computing, a node can be a physical computer, a virtual computer, referred to as a VM, or even a container.
क्लाउड कंप्यूटिंग में, नोड एक भौतिक कंप्यूटर, एक वर्चुअल कंप्यूटर, जिसे वर्चुअल (Virtual Machine) मशीन कहा जा सकता है या एक कंटेनर भी हो सकता है।

now it's better. google translate sometimes jumbles up the words.